### PR TITLE
Add server api for my-dev-server

### PR DESCRIPTION
**Pull request by IlyaMenkov**
Added ability to create, get and delete user

### CODE REVIEW

File: etc/my-dev-server/my-dev-server.conf [hunk 1]
```diff
@@ -1,2 +1,3 @@
 [DEFAULT]
-db_url=mysql+pymysql://root@localhost/my_dev
+db_url=mysql+pymysql://root:r00tme@localhost/my_dev
```
Comment (esikachev) [inline etc/my-dev-server/my-dev-server.conf]: you need to remove password

File: my_dev_server/config.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 db_opts = [
     cfg.StrOpt('db_url',
-               default='mysql+pymysql://root@localhost/my_dev',
+               default='mysql+pymysql://root:r00tme@localhost/my_dev',
```
Comment (esikachev) [inline my_dev_server/config.py]: ditto

File: my_dev_server/db/models.py [hunk 3]
```diff
@@ -38,4 +43,9 @@ def __init__(self, user_id=None, alias=None, host=None, username=None,
         self.password = password
 
     def __repr__(self):
-        return '<SSH %r>' % (self.id)
\ No newline at end of file
+        return '<SSH %r>' % (self.id)
+
+    def to_json(self):
+        return dict(user_id=self.user_id,
```
Comment (esikachev) [inline my_dev_server/db/models.py]: where is other parameters?

File: my_dev_server/server/app.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
+
```
Comment (esikachev) [inline my_dev_server/server/app.py]: please, remove

File: my_dev_server/server/app.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+from my_dev_server.db.models import User, Ssh
```
Comment (esikachev) [inline my_dev_server/server/app.py]: let's import only modules

File: my_dev_server/server/app.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+from flask import Flask, request, jsonify
```
Comment (esikachev) [inline my_dev_server/server/app.py]: new line for new import

File: requirements.txt [hunk 7]
```diff
@@ -7,4 +7,6 @@ pbr>=1.6 # Apache-2.0
 pymysql
 SQLAlchemy<1.1.0,>=1.0.10 # MIT
 oslo.config
-oslo.log
\ No newline at end of file
+oslo.log
```
Comment (esikachev) [inline requirements.txt]: let's remove this line?

File: my_dev_server/db/models.py (hunk 3 above)
Comment (esikachev) [inline my_dev_server/db/models.py]: why this is not fixed?

Comment (esikachev) [inline my_dev_server/db/models.py]: all username, passsword

**Comment by esikachev** [comment]
@IlyaMenkov please, rebase manually

File: my_dev_server/server/main.py [hunk 8]
```diff
@@ -51,20 +51,21 @@ def user_create():
     return user_json
 
 
-@mydev.route('/users/<id>', methods=['GET'])
+@mydev.route('/users/<id>', methods=['GET'], strict_slashes=False)
 def user_get(id):
     LOG.info('%s %s' % (request.method, id))
     user = models.User.query.filter_by(id=id).first()
```
Comment (IlyaMenkov) [inline my_dev_server/server/main.py]: If user with id does'not exit, return will be failed

File: my_dev_server/server/main.py [hunk 9]
```diff
@@ -51,20 +51,21 @@ def user_create():
...
 def user_delete(id):
     LOG.info('%s %s' % (request.method, id))
     models.User.query.filter_by(id=id).delete()
     base.session.commit()
+    return '200'
 
 
 def main():
-    mydev.run(debug=True)
+    mydev.run(debug=True, host='0.0.0.0')
```
Comment (IlyaMenkov) [inline my_dev_server/server/main.py]: I think that we should configure debug for developer and customer servers

Comment (IlyaMenkov) [inline my_dev_server/server/main.py]: should we configure host?

**Pull request merged**